### PR TITLE
OSMT 3.0 Changelog

### PR DESCRIPTION
**Pull request by Corpratespaz**
View file as markdown here:
https://github.com/wgu-opensource/osmt/blob/9fef8b9442f439de179d18eaae28f8abaa48af52/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,82 @@
...
+
+- New API test suite
+
+## Model Changes:
+- RSD 'category' -> 'categories'
+
+- RSD 'author' -> 'authors'
+
+## Defect Fixes
+- Removed leading `'-'` character from URLs in export files
```
Comment (JohnKallies) [inline CHANGELOG.md:36]: This was only in the "dev" Spring profile, FWIW...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+
+## Removed
+- Protractor e2e (end-to-end) testing framework (deprecated & unused by OSMT)
```
Comment (manuel-delvillar) [inline CHANGELOG.md:39]: TSLint & codelyzer were also removed, we can add that only if we want to be more detailed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+- Skills (RSDs) now support multiple authors
+
+- Search result filtering
+
+- Selectable pagination quantity
+
+- Export to Excel Workbook .xlsx option (previously only CSV)
+
+## Updates
+
```
Comment (manuel-delvillar) [inline CHANGELOG.md:19]: New dependencies added related to `ESLint`

**Comment by JohnKallies** [comment]
@Corpratespaz got us started with a great 1st draft here. I added some additional details from @psanchez-wgu. Still looking for an approval from the OSE team.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,79 @@
...
+## Updates
+- Updated UI to Angular v16 (previously v12)
+- Updated other dependencies
+
+## Developer/Integrator Enhancements
+- OpenAPI compliant API specification
+- New API test suite
+- Full support for loading and reindexing a static development dataset via `osmt_cli.sh`
+
+## Model Changes:
```
Comment (wgu-edwin) [inline CHANGELOG.md:25]: This header has a colon while the others do not. Could be a typo?

**Review by wgu-edwin — CHANGES_REQUESTED** [review]

**Review by wgu-edwin — APPROVED** [review]
Looks good to me

**Review by psanchez-wgu — APPROVED** [review]

**Pull request merged**